README.md

```diff
@@ -44,8 +44,10 @@
 DataFusion is an extensible query engine written in [Rust] that
 uses [Apache Arrow] as its in-memory format.
 
-The DataFusion libraries in this repository are used to build data-centric system software. DataFusion also provides the
-following subprojects, which are packaged versions of DataFusion intended for end users.
+The DataFusion project provides libraries and binaries for data system
+developers and end users such as data engineers and analysts. This repository
+contains libraries for system developers. The following related projects work
+out of the box and target end users.
```

Comment:
The DataFrame API that this project ships, for whom it is? (mentioned 13 lines below)

Also, "The target audience for the DataFusion crates in this repository are developers" (9 lines below) sounds like a repetition now.

Reply:
> The DataFrame API that this project ships, for whom it is? (mentioned 13 lines below)

It is a good question -- I think @timsaucer and @Omega359 uses the Rust DataFrame library as is.

Reply:
I do use the dataframe api almost exclusively. I say almost because I do allow sql expressions in my library which are converted to Expr's in DF and used via the dataframe api.

Reply:
I do use the DataFrame API exclusively, but I work almost entirely via `datafusion-python`.